tools/oecert/host/host.cpp

```diff
@@ -397,8 +481,9 @@ static int _parse_args(int argc, const char* argv[])
         return 1;
     }
 
-    if (_params.generate_certificate && _params.generate_report &&
-        _params.generate_evidence)
+    if (_params.generate_certificate && _params.generate_legacy_report_remote &&
+        _params.generate_sgx_ecdsa && _params.generate_sgx_epid_linkable &&
+        _params.generate_sgx_epid_unlinkable)
```

Comment:
```suggestion
    if (_params.generate_certificate + _params.generate_legacy_report_remote +
        _params.generate_sgx_ecdsa + _params.generate_sgx_epid_linkable +
        _params.generate_sgx_epid_unlinkable != 1)
```
In case user didn't give `--format` input at all or give more than one formats. For example, current `./oecert --verify` will output nothing and didn't trigger `_display_help()`. And current `./oecert --evidence --report` will generate a report as program check `_params.generate_report` before `_params.generate_evidence`. Could you help fix it in this PR?